docs/interpreter/livy.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "Livy Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+## Livy Interpreter for Apache Zeppelin
+Livy is an open source REST interface for interacting with Spark from anywhere. It supports executing snippets of code or programs in a Spark context that runs locally or in YARN.
+
+* Interactive Scala, Python and R shells
+* Batch submissions in Scala, Java, Python
+* Multi users can share the same server (impersonation support)
```

Comment:
Added a small note for the same https://github.com/apache/incubator-zeppelin/pull/827/commits/53f280462759517e3f4329f35b83640e79584715